e2etest-functions/simple-chat/csharpv3/local.settings.sample.json

```diff
@@ -0,0 +1,13 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "AzureSignalRConnectionString": "<signalr-connection-string>",
+    "AzureSignalRConnectionString:Backup:Secondary": "<>"
```

Comment:
Since this is for tests, move them to test folder.

Reply:
Also no need to make this sample.json but directly local.settings.json as this is for tests.